src/main/scala/codechicken/core/launch/DepLoader.java

```diff
@@ -501,7 +501,7 @@ private void scanDepInfos() {
         private void scanDepInfo(File file) {
             try {
                 ZipFile zip = new ZipFile(file);
-                ZipEntry e = zip.getEntry("dependancies.info");
```

Comment:
Renaming might be an issue as well since other mods do use it as well with .json. Just removed this on another codechicken mod. Leave it open for one or two more weeks and we might be able to remove it completely. Gonna strip dependencies out all I can.